### PR TITLE
add a policy for showing integrity shield deny event in grc ui

### PR DESCRIPTION
**Pull request by hirokuni-kitahara**
- fix for the issue https://github.com/open-cluster-management/backlog/issues/8500 , UI will show `NonCompliant` for the added policy when deny event has been reported in managed cluster

### CODE REVIEW

**Comment by ycao56** [comment]
@hirokuni-kitahara can you add this policy to readme? https://github.com/open-cluster-management/policy-collection/tree/master/community

**Comment by hirokuni-kitahara** [comment]
@ycao56 yes, I will update a corresponding document in integrity shield repo first, then add README change to this PR. I will let you know when they are ready. thank you.

**Comment by hirokuni-kitahara** [comment]
hi @ycao56 , I have updated community/README.md in this PR with the link to integrity shield document. thank you.

**Review by ycao56 — APPROVED** [review]

**Comment by ycao56** [comment]
looks good to me. @dockerymick can you review the texts?

**Review by tphee — APPROVED** [review]

File: community/CM-Configuration-Management/policy-integrity-shield-events.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+apiVersion: policy.open-cluster-management.io/v1
```
Comment (dockerymick) [inline community/CM-Configuration-Management/policy-integrity-shield-events.yaml:6]: I recommend adding a brief comment description of the policy functionality. It is not required, but can be helpful for the user, similar to the community policy: [policy-compliance-operator-install-upstream](https://github.com/open-cluster-management/policy-collection/blob/master/community/CA-Security-Assessment-and-Authorization/policy-compliance-operator-install-upstream.yaml)

**Review by dockerymick — APPROVED** [review]
I made a small punctual update for the description that is in the README. I also left a suggestion for the policy file. Otherwise this PR looks good to me

File: community/CM-Configuration-Management/policy-integrity-shield-events.yaml (hunk 1 above)
Comment (hirokuni-kitahara) [reply community/CM-Configuration-Management/policy-integrity-shield-events.yaml:6]: thank you @dockerymick , I have added a description to the policy.

**Comment by hirokuni-kitahara** [comment]
@dockerymick Thank you, I have updated the yaml and have added the comment description.
@gparvin I think all reviews are ok, and could you help me to merge this PR if possible? Thank you.

**Comment by gparvin** [comment]
This has been merged.  Thanks for contributing this policy!

**Pull request merged**